### PR TITLE
Port GeoRSS to mobx

### PR DESCRIPTION
**Pull request by zoran995**
### What this PR does

Port GeoRSS from master to mobx, also added support to skip items without geometry (not sure if that is considered to be valid GeoRSS file but it happens with FAO geonetwork GeoRSS response).

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by tephenavies** [comment]
I'm going to review and merge this after we get #4466 merged (which should be before the end of the week). Thanks for the contribution!

**Review by tephenavies — CHANGES_REQUESTED** [review]

File: lib/Models/GeoRssCatalogItem.ts
Comment (zoran995) [reply lib/Models/GeoRssCatalogItem.ts]: Should be fixed now. Conflicts with mobx branch are resolved

**Comment by tephenavies** [comment]
Tests passing here: https://travis-ci.org/github/TerriaJS/terriajs/builds/706445766

Thanks for the PR!

**Review by tephenavies — APPROVED** [review]

**Pull request merged**
